.github/workflows/pr_notifier.yml

```diff
@@ -3,6 +3,9 @@ on:
   schedule:
   - cron: '0 5 * * 1,2,3,4,5'
 
+permissions:
+  contents: read
```

Comment:
in this case i think we want to remove all permissions - ie `permissions: {}`

Reply:
Removed.